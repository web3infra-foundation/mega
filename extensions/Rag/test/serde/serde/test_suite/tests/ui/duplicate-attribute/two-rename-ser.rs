use serde_derive::Serialize;

#[derive(Serialize)]
struct S {
    #[serde(rename(serialize = "x"))]
    #[serde(rename(serialize = "y"))]
    x: (),
}

fn main() {}
