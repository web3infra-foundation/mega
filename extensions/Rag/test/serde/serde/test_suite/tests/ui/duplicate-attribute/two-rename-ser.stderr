error: duplicate serde attribute `rename`
 --> tests/ui/duplicate-attribute/two-rename-ser.rs:6:13
  |
6 |     #[serde(rename(serialize = "y"))]
  |             ^^^^^^
